server.js

```diff
@@ -0,0 +1,78 @@
+const { client, syncAndSeed } = require ('./db')
+const express = require ('express')
+const path = require ('path')
+
+const app = express()
+
+app.use('/assets', express.static(path.join(__dirname, 'assets')))
+
+app.get('/', async (req, res, next) => {
```

Comment:
i think this route should be `/books` to keep consistent with the route on line 38
that means you'd need a redirect route such as ```app.get('/', (req, res) => res.redirect('/books'))` to make sure you cant go to `/`